contact[1].php

```diff
@@ -1,5 +1,15 @@
+<?php
+	session_start();
```

Comment:
PHP code that brings the user back to the login page if they haven't signed in yet